src/qt/radiobut.cpp

```diff
@@ -42,7 +44,18 @@ bool wxRadioButton::Create( wxWindow *parent,
     m_qtRadioButton = new QRadioButton( parent->GetHandle() );
     m_qtRadioButton->setText( wxQtConvertString( label ));
 
-    return QtCreateControl( parent, id, pos, size, style, validator, name );
+    bool createdOk = QtCreateControl( parent, id, pos, size, style, validator, name );
+
+    if( ( style & wxRB_GROUP ) || ( style & wxRB_SINGLE ) )
```

Comment:
```suggestion
    if ( (style & wxRB_GROUP) || (style & wxRB_SINGLE) )
```
...and many other odd spaces.

Reply:
I'll take a look now.